README.md

```diff
@@ -2,7 +2,7 @@
   <img src="https://github.com/icerpc/icerpc-csharp/raw/main/.github/assets/icerpc-banner.svg" height="100" width="100" />
 </p>
 
-# IceRPC for C#
```

Comment:
This is something we should be doing in all of our README.md when we have `C#`. Most markdown linters will complain about it.